include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Ok, nevermind that idea then :)

Anyways, I still suggest to remove the #ifdef even though the error is much less clear. Reason is that not defining USE_LONG_LONG is quite likely, whereas calling the long long assert on a system that doesn't support long long is rather unlikely. So, to get the best default, the #ifdef can be removed.